plugins/available/z_autoenv.plugin.bash

```diff
@@ -37,7 +36,9 @@ cd() {
 		autoenv_init
 		return 0
 	else
+		local return_code
+		return_code=$?
```

Comment:
This doesn't work. `local return_code` would overwrite `$?`, so the exit status of `builtin cd "$@"` is lost.